Add PubWise Analytics

GLStephen opened the pull request:
## Type of change
- [ X ] Other - Analytics Adapter

## Description of change
This is the analytics adapter for PubWise.io

- stephenj@pubwise.io
- [ x ] official adapter submission

GLStephen commented:
The build failure looks unrelated.

matthewlane commented on src/adapters/analytics/pubwiseanalytics.js:
```diff
@@ -0,0 +1,66 @@
+      if (args !== undefined && args.config !== undefined) {
+        target_site = args.config.site;
+      }
+      if (args !== undefined && args.config !== undefined) {
```
This line is the same as line 43, is that on purpose? When running tests on this PR there are a few `WARN [web-server]: 404: /undefined` warnings in the console, which may be coming from line 47. Adding ` && args.config.endpoint` before the closing `)` on 46 stops those warnings, but I may be misunderstanding the intention here

GLStephen replied:
The config, and sometimes args, is not available in future calls of _track after the initial call of enable analytics. Overriding/extending enable led to other issues specifically multiple analytics adapters would no longer load. Since I have not been able to fix either of those other issues globally this was my compromise. I will clean up the logic here and look into the WARN [web-server]: 404: /undefined issue.

GLStephen commented on src/adapters/analytics/pubwiseanalytics.js:
```diff
@@ -0,0 +1,66 @@
+
+export default pubwiseAnalytics;
+
+
```
I'll remove that when I fix the above.

GLStephen commented:
I have updated the analytics adapter based on feedback as well as removed a bit of event filtering since we've updated the endpoint to handle determination of event handling at the API endpoint.

GLStephen commented:
Downloader addition is here https://github.com/prebid/prebid.github.io/pull/242 Thanks!

GLStephen commented:
@matthewlane @protonate Any idea what version this might land in? 24? Just trying to plan ahead. Thanks!

mkendall07 commented:
LGTM.
